crates/vm/levm/src/constants.rs

```diff
@@ -1,5 +1,7 @@
 use ethereum_rust_core::U256;
 
+use crate::errors::{InternalError, VMError};
```

Comment:
Let's try to keep the imports together.
```suggestion
use crate::errors::{InternalError, VMError};
```